userguide/platform/userguide_configuration.adoc

```diff
@@ -103,283 +104,142 @@ This sets `org.killbill.invoice.dryRunNotificationSchedule=48h` for the `boblaza
+|Database   |org.killbill.dao.cachePrepStmts   |KB_org_killbill_dao_cachePrepStmts   |Enable prepared statements cache   |true   |Config File/Environment Variable   
+|Database   |org.killbill.dao.useServerPrepStmts   |KB_org_killbill_dao+useServerPrepStmts   |Enable server-side prepared statements   |true   |Config File/Environment Variable   
+|Database   |org.killbill.dao.dataSourceClassName   |KB_org_killbill_dao_dataSourceClassName   |DataSource class name provided by the JDBC driver, leave null for autodetection   |null   |Config File/Environment Variable   
+|Database   |org.killbill.dao.driverClassName   |KB_org_killbill_dao_driverClassName   |JDBC driver to use (when dataSourceClassName is null)   |null   |Config File/Environment Variable    
+|Database   |org.killbill.dao.mysqlServerVersion   |KB_org_killbill_dao_mysqlServerVersion   |MySQL server version   |5.1   |Config File/Environment Variable     
+|Database   |org.killbill.dao.logLevel   |KB_org_killbill_dao_logLevel   |Log level for SQL queries   |DEBUG   |Config File/Environment Variable   
+|Database   |org.killbill.dao.poolingType   |KB_org_killbill_dao_poolingType   |Connection pooling type   |HIKARICP   |Config File/Environment Variable    
+|Database   |org.killbill.dao.healthCheckConnectionTimeout   |KB_org_killbill_dao_healthCheckConnectionTimeout   |How long to wait before a connection attempt to the database is considered timed out (healthcheck only)   |10s   |Config File/Environment Variable   
+|Database   |org.killbill.dao.healthCheckExpected99thPercentile   |KB_org_killbill_dao_healthCheckExpected99thPercentile   |Expected 99th percentile calculation to obtain a connection (healthcheck only)   |50ms   |Config File/Environment Variable    
+|Database   |org.killbill.dao.initializationFailFast   |KB_org_killbill_dao_initializationFailFast   |Whether or not initialization should fail on error immediately   |false   |Config File/Environment Variable    
+|Database   |org.killbill.dao.transactionIsolationLevel   |KB_org_killbill_dao_transactionIsolationLevel   |Set the default transaction isolation level   |TRANSACTION_READ_COMMITTED   |Config File/Environment Variable   
+|Database   |org.killbill.dao.readOnly   |KB_org_killbill_dao_readOnly   |Whether to put connections in read-only mode   |false   |Config File/Environment Variable   
+|Cache   |org.killbill.catalog.frequentValuesCacheSize   |-   |Coming Soon   |1000   |System property 
+|Security   |killbill.server.ldap   |-   |Coming Soon   |false   |System property   
+|Security   |killbill.server.okta   |-   |Coming Soon   |false   |System property    
+|Security   |killbill.server.auth0   |-   |Coming Soon   |false   |System Property   
+|Security   |killbill.server.rbac   |-   |Coming Soon    |true   |System Property   
+|===
```

Comment:
I've added the system properties by searching for `System.getProperty` calls in the Kill Bill code.